README.md

```diff
@@ -44,31 +44,49 @@
 
    Your site is now running at `http://localhost:8000`!
 
-    *Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://next.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).*
+   \_Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://next.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).\_
```

Comment:
nit: I think using the * looks better over having _Note.

Reply:
This is also incorrect because the escaped `_` mean it is not longer emphasized text

Reply:
How about putting it in quotes like this:

> **Note:** You'll also see...

Reply:
I think the format is coming from the upstream Gatsby boilerplate, so I don't know if it makes sense to change it